icons/badge-dollar-sign.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
+  <path d="M16 8h-6a2 2 0 0 0-2 2v0a2 2 0 0 0 2 2h4a2 2 0 0 1 2 2v0a2 2 0 0 1-2 2H8" />
+  <path d="M12 18V6" />
```

Comment:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
  <path d="M16 8h-6a2 2 0 1 0 0 4h4a2 2 0 1 1 0 4H8" />
  <path d="M12 18V6" />
```

Reply:
We could also make the dollar smaller `<path d="M15 8h-4.5a2 2 0 1 0 0 4h3a2 2 0 1 1 0 4H9" />` ![file-question](https://user-images.githubusercontent.com/25524993/233153579-28e5d3ca-1aec-4847-839a-20dcecf6fbde.svg)